### ISSUE TITLE
Unexpected result for an object-like variable

### ISSUE THREAD

**Issue by achou11** [peterpeterparker/tsdoc-markdown#44]
Given a source file containing the following:

```ts
export const MyObject = {
	someField: 'foo',
	someFunction: () => {},
}
```

I get the following when running `tsdoc-markdown@v1.0.0` with the defaults:

```md
## :toolbox: Functions

- [someFunction](#gear-somefunction)

### :gear: someFunction

| Function | Type |
| ---------- | ---------- |
| `someFunction` | `() => void` |
```

Is this a known limitation of functionality? Found it unexpected that `someFunction()` is extracted out, without any mention of the context it lives in (`MyObject`).

Not exactly sure what the exact output I'm expecting is, but I would think `MyObject` is documented somehow such that there's mention of its fields (`someField` and `someFunction`).

**Comment by peterpeterparker** [comment]
Good point. I wouldn't say it's a known limitation, but rather a feature that hasn't been implemented yet. At least, there isn't such a use case in the mock data for the tests. I agree that the context should be preserved in some way.

**Comment by peterpeterparker** [comment]
Thanks. I've implemented what feels like the simplest approach: providing some context from the parent to the inner functions.

For example:

```
### :gear: MyObject.someFunction

| Function | Type |
| ---------- | ---------- |
| `MyObject.someFunction` | `() => void` |
```

There could be a more complex solution where the object itself becomes a chapter in the documentation, but that seems like a significant amount of work, and I personally don't have a need for it at the moment. However, if you or anyone else is interested in submitting a PR to enhance this, of course this is welcomed.

The feature was released in v1.1.0 this evening.

Thanks for sharing the issue!
